src/server/verification/verificationAPI.js

```diff
@@ -45,7 +45,10 @@ const setup = (app: Router, verifier: VerificationAPI, gunPublic: StorageAPI, st
         await processor.enqueueDisposal(user, enrollmentIdentifier, log)
       } catch (exception) {
         const { message } = exception
-        logException(log, 'delete face record failed:', message, exception, { enrollmentIdentifier, user })
+        notifyAboutVerificationIssue(log, 'delete face record failed:', message, exception, {
```

Comment:
change it to if(shouldLogVerificaitonError()) log.error else log.warn

Reply:
also i dont understand why it can be a warn and not always an error? if deletion failed it should be an error

Reply:
all possible issues here:
- invalid signature passed (e.g. invalid input)
- mongodb failure (unable to add record to the collection)
- wallet failure (failed to call removeWhitelisted)


should all them to be logged as errors ?

Reply:
seems like yes to me